.github/workflows/ironpython_linux.yml

```
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Linux_Ironpython_UnitTests
+
+
+# Controls when the workflow will run
+on:
+  workflow_dispatch:
+  schedule:  # UTC at 0400
+    - cron:  '0 4 * * *'
+   
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
```

Giving the job a better name helps with finding the right action in [Actions](https://github.com/pyansys/PyAEDT/actions).

```suggestion
  linux_iron_python:
  - name IronPython Unit Tests
```